freqgen/api.py

```diff
@@ -99,3 +108,9 @@ def predict(
         playlist=PlaylistLinks(**model.get_best_playlist(best_station)),
         image=image,
     )
+
+
+@app.get("/analytics/all")
+def get_analytics():
```

Comment:
Ici idéalement il faut typer : 

```suggestion
class Analytics(BaseModel):
    questionnaire_completed: int

def get_analytics() -> Analytics:
```